frontend/src/app/signUp/IdentityVerification/Consent.test.tsx

```diff
@@ -3,12 +3,23 @@ import { MemoryRouter } from "react-router";
 
 import Consent from "./Consent";
 
+jest.mock("react-router-dom", () => ({
+  Redirect: (props: any) => `Redirected to ${props.to.pathname}`,
```

Comment:
Whoa this is a really cool way to test redirects, I dig it!

Reply:
hahah yeah its basically testing the logic of redirecting without actually redirecting, credit stackoverflow